### ISSUE TITLE
F63 shouldn't indicate use of preceding heading to meet 2.4.4

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag#819]
I believe we determined that the preceding heading was not sufficient to meet 2.4.4 and thus we should remove as way to pass F63.

**Comment by alastc** [comment]
So the change would be to remove the heading bullet in the description, and step one of the procedure?
https://www.w3.org/WAI/WCAG21/Techniques/failures/F63

**Comment by mraccess77** [comment]
Yes

**Comment by alastc** [comment]
Ok, PR created. Do you happen to know when that was decided? (I don't remember that.) Just in case someone queries.

**Comment by mraccess77** [comment]
I don't recall the date -- but it was several years ago - maybe 2016?

**Comment by patrickhlauke** [comment]
As @scottaohara reminded me today...how does this stack up against still having H80 "Identifying the purpose of a link using link text combined with the preceding heading element" ? https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/H80
